kotlin/src/main/com/looker/rtl/Constants.kt

```diff
@@ -6,13 +6,18 @@ import org.jetbrains.annotations.NotNull
 const val LOOKER_VERSION = "6.21"
 const val API_VERSION = "3.1"
 const val SDK_VERSION = "${API_VERSION}.${LOOKER_VERSION}"
+const val AGENT_TAG = "SDK-KT ${SDK_VERSION}"
```

Comment:
This is slightly different than what we did in the TS and Python implementations, where the agent tag was of the form `LANG-SDK {SDK_VERSION}`. 
```suggestion
const val AGENT_TAG = "KT-SDK ${SDK_VERSION}"
```

Reply:
I had a conversation with @dthorpe this weekend about how to tag the SDKs. We're going to end up with something like:
`Looker SDK-XX 3.1.6.24` where `XX` is the platform specifier, `3.1` is the SDK version, and `6.24` is the Looker release version the SDK was generated from.
We may support additional attributes to identify the "app" making an API call, but that's our minimum.